src/Compilers/CSharp/Test/Syntax/Parsing/LineSpanDirectiveParsingTests.cs

```diff
@@ -275,6 +307,51 @@ public void LineDirective_06()
         {
             string source = @"#line(1,2)-(3,4)5""file.cs""";
 
+            UsingLineDirective(source, options: null,
+                // (1,6): error CS9028: The #line span directive requires space before the first parenthesis, before the character offset and before the file name
+                // #line(1,2)-(3,4)5"file.cs"
+                Diagnostic(ErrorCode.ERR_LineSpanDirectiveRequiresSpace, "(1,2)").WithLocation(1, 6),
+                // (1,17): error CS9028: The #line span directive requires space before the first parenthesis, before the character offset and before the file name
+                // #line(1,2)-(3,4)5"file.cs"
+                Diagnostic(ErrorCode.ERR_LineSpanDirectiveRequiresSpace, "5").WithLocation(1, 17),
+                // (1,18): error CS9028: The #line span directive requires space before the first parenthesis, before the character offset and before the file name
+                // #line(1,2)-(3,4)5"file.cs"
+                Diagnostic(ErrorCode.ERR_LineSpanDirectiveRequiresSpace, @"""file.cs""").WithLocation(1, 18)
+                );
+
+            N(SyntaxKind.LineSpanDirectiveTrivia);
+            {
+                N(SyntaxKind.HashToken);
+                N(SyntaxKind.LineKeyword);
+                N(SyntaxKind.LineDirectivePosition);
+                {
+                    N(SyntaxKind.OpenParenToken);
+                    N(SyntaxKind.NumericLiteralToken, "1");
+                    N(SyntaxKind.CommaToken);
+                    N(SyntaxKind.NumericLiteralToken, "2");
+                    N(SyntaxKind.CloseParenToken);
+                }
+                N(SyntaxKind.MinusToken);
+                N(SyntaxKind.LineDirectivePosition);
+                {
+                    N(SyntaxKind.OpenParenToken);
+                    N(SyntaxKind.NumericLiteralToken, "3");
+                    N(SyntaxKind.CommaToken);
+                    N(SyntaxKind.NumericLiteralToken, "4");
+                    N(SyntaxKind.CloseParenToken);
+                }
+                N(SyntaxKind.NumericLiteralToken, "5");
+                N(SyntaxKind.StringLiteralToken, "\"file.cs\"");
+                N(SyntaxKind.EndOfDirectiveToken);
+            }
+            EOF();
+        }
+
+        [Fact, WorkItem(61663, "https://github.com/dotnet/roslyn/issues/61663")]
```

Comment:
>WorkItem(61663, "https://github.com/dotnet/roslyn/issues/61663") [](http://example.com/codeflow?start=15&length=64)

It feels like the `[WorkItem]` attribute belongs on the previous test rather than this one.